### PR TITLE
package_r_crisprseek biocundoctor dependancy definitions added

### PR DESCRIPTION
**Pull request by nephantes**
I am going to port CRISPRseek to galaxy. Could you please check and merge my CRISPRseek dependancy.

### CODE REVIEW

File: packages/package_r_crisprseek_1_10_0/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="R" version="3.2.1">
+        <repository name="package_r_3_2.1" owner="iuc" prior_installation_required="True" />
```
Comment (erasche) [inline packages/package_r_crisprseek_1_10_0/tool_dependencies.xml]: should be `package_r_3_2_1`

File: packages/package_r_crisprseek_1_10_0/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <package name="crisprseek" version="1.10.0">
+        <install version="1.0">
+            <actions>
+                <action type="setup_r_environment">
+                    <repository name="package_r_3_2_1" owner="iuc">
+                        <package name="R" version="3.2.1" />
+                    </repository>
+                    
+                    <!-- bioconductor package-->
+                    <package>http://www.bioconductor.org/packages/release/bioc/src/contrib/CRISPRseek_1.10.0.tar.gz</package>
```
Comment (erasche) [inline packages/package_r_crisprseek_1_10_0/tool_dependencies.xml]: We use the galaxy project bioarchive for permanent copies of bioC packages.

https://bioarchive.galaxyproject.org/CRISPRseek_1.11.0.tar.gz

This package also looks like it has a few other dependencies: https://bioarchive.galaxyproject.org/#/pkg/CRISPRseek/1.11.0/ judging by the `Imports` section.

**Comment by erasche** [comment]
@nephantes thanks for the contribution! Only a couple small things (you seem to have patched the R version already) and we'll get this merged for you.

I'll update the `.shed.yml` owner and URL after merge.

File: packages/package_r_crisprseek_1_10_0/tool_dependencies.xml (hunk 2 above)
Comment (erasche) [inline packages/package_r_crisprseek_1_10_0/tool_dependencies.xml]: Dependencies seem to be:

- https://bioarchive.galaxyproject.org/BiocGenerics_0.17.0.tar.gz
- https://bioarchive.galaxyproject.org/Biostrings_2.39.0.tar.gz
- https://bioarchive.galaxyproject.org/IRanges_2.5.4.tar.gz

Comment (nephantes) [inline packages/package_r_crisprseek_1_10_0/tool_dependencies.xml]: How add optional dependancies like below?
BSgenome.Hsapiens.UCSC.hg19, TxDb.Hsapiens.UCSC.hg19.knownGene, org.Hs.eg.db

Comment (erasche) [inline packages/package_r_crisprseek_1_10_0/tool_dependencies.xml]: Just add the URLs I listed in the comment above, that should get you the BSgenome data. As for the others... I'm actually not sure how to get those out of bioconductor easily. They aren't part of the bioconductor SVN server, we (the IUC) will have to figure that out and update the package when we have a solution.

Comment (nephantes) [inline packages/package_r_crisprseek_1_10_0/tool_dependencies.xml]: Great! Let me know, when it is ready.

**Comment by erasche** [comment]
@nephantes the extra data files won't be something we can get done immediately, but we can update the package later so I'd like to go ahead and get this merged now so you can start making use of it. 

Last question, do you need both a 1.10 and 1.11 version? If not I'd like to just add the more recent 1.11 version.

**Comment by nephantes** [comment]
No I don't need 1.10. Latest available is fine. Thanks!!!

**Comment by erasche** [comment]
Gerat, thanks. I'll get this merged and the last bits touched up. Thanks again for the contribution @nephantes

**Pull request merged**